nf_core/pipeline-template/tests/modules/samplesheet_check/main.nf.test

```diff
@@ -1,15 +1,14 @@
 nextflow_process {
-    name "samplesheet_check"
+
+    name "Test Process SAMPLESHEET_CHECK"
```

Comment:
I always feel redundant writing "Test" in a test. We just need to come to a community consensus on typical test names.

Reply:
i was trying to follow the convention of `nf-test generate` cli command but, yes this is a Q for the community

Reply:
Oh man, I haven't been using nf-test generate even :see_no_evil: 

I think that makes more sense then!